docs/developer-docs/latest/plugins/i18n.md

```diff
@@ -237,39 +267,47 @@ This request:
 
 ::: response Example response
 
+<!-- TODO: check example reponse -->
```

Comment:
This is correct :)

Reply:
![image](https://user-images.githubusercontent.com/8593673/139721269-3e937ad0-169b-4c59-9f72-5de037e1d4e4.png)